src/components/autocomplete/js/autocompleteController.js

```diff
@@ -208,6 +208,17 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
   }
 
   /**
+   * Find the target for the wrap element which determines menu width
```

Comment:
while the name `warp` describes what the element is, as far as the controller is concerned it only uses the wrap to calculate the width which can be found in `positionDropdown ()`. I could change the comment to something like "Find the target for the wrap element." But I thought more description help convey why this code is there. Again the description is similar to that of `getSnapTarget()`.